src/python/pants/backend/python/goals/export_test.py

```diff
@@ -127,13 +129,29 @@ def test_export_venv_old_codepath(
 
     reldirs = [result.reldir for result in all_results]
     if enable_resolves:
-        assert reldirs == [
-            "python/virtualenvs/a",
-            "python/virtualenvs/b",
-            "python/virtualenvs/tools/flake8",
-        ]
+        if py_resolve_format == PythonResolveExportFormat.symlinked_immutable_virtualenv:
+            assert reldirs == [
+                f"python/virtualenvs/a/{current_interpreter}",
+                f"python/virtualenvs/b/{current_interpreter}",
+                f"python/virtualenvs/tools/flake8/{current_interpreter}",
+            ]
+        else:
+            assert reldirs == [
+                f"python/virtualenvs/a/{current_interpreter}",
+                f"python/virtualenvs/b/{current_interpreter}",
+                "python/virtualenvs/tools/flake8",
+            ]
     else:
-        assert reldirs == ["python/virtualenv", "python/virtualenvs/tools/flake8"]
+        if py_resolve_format == PythonResolveExportFormat.symlinked_immutable_virtualenv:
+            assert reldirs == [
+                f"python/virtualenv/{current_interpreter}",
+                f"python/virtualenvs/tools/flake8/{current_interpreter}",
```

Comment:
Why is the interpreter present for tool venvs in the symlink case, but not the other? I would expect it to be consistent across both cases

Reply:
Annoying historical reasons. I want it to be present in all cases, but this requires a deprecation cycle because people may be relying on it not being there for tool venvs in the non-symlink case, because that was the preexisting behavior.

Reply:
Ah ok, I'd missed the previous change so thought it was new behavior as of this PR